src/libraries/sendtohelixhelp.proj

```diff
@@ -240,9 +240,23 @@
     </ItemGroup>
   </Target>
 
+  <!-- XUnitLogChecker required configuration -->
   <ItemGroup Condition="Exists('$(XUnitLogCheckerLibrariesOutDir)')">
     <HelixCorrelationPayload Include="$(XUnitLogCheckerLibrariesOutDir)" />
-  </ItemGroup>  
+    <HelixCorrelationPayload Condition="'$(WindowsShell)' == 'true'" Include="dotnet-sos">
+      <Destination>sos</Destination>
+      <Uri>https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/flat2/dotnet-sos/$(DotnetSosVersion)/dotnet-sos.$(DotnetSosVersion).nupkg</Uri>
+    </HelixCorrelationPayload>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetOS)' == 'windows'">
+    <SosPreCommand Include="set _NT_SYMBOL_PATH=%HELIX_CORRELATION_PAYLOAD%;%HELIX_CORRELATION_PAYLOAD%\shared\Microsoft.NETCore.App\9.0.0" />
```

Comment:
Can we avoid hardcoding the version here?  Seems like we should have a property with that available.  Do we also need to have the work item directory added to the symbol path.  I think the test PDBs and its dependencies are there.

Honestly I'm surprised this is required since the dump came from the same machine the debugger is running on, the module paths will be in the dump, and the symbols are next to the modules.  @hoyosjs what did you see that made you realize the debugger couldn't find the symbols?

Reply:
> Do we also need to have the work item directory added to the symbol path. I think the test PDBs and its dependencies are there.

If yes, my assumption would be that the variable would be %HELIX_DUMP_FOLDER%, but I might be wrong.

Reply:
Interestingly, the CoreCLR tests do not use the same value. Instead, they use:

https://github.com/dotnet/runtime/blob/00874d6fdec31ec8ffec4d05b18eb849f1d4f1a0/src/tests/Common/helixpublishwitharcade.proj#L643

Reply:
> Honestly I'm surprised this is required since the dump came from the same machine the debugger is running on, the module paths will be in the dump, and the symbols are next to the modules

I know VS has such symbol heuristic. I don't think windbg has that. It does probe the path in the debug directory of the dll, but that's only relevant on the build machine. 

> @hoyosjs what did you see that made you realize the debugger couldn't find the symbols?

I had a hunch, `!sym noisy` confirms the probing behavior, and I tested locally with the helix payload. After that the symbols resolve correctly.

Reply:
Marking this as resolved, I addressed the suggestions. The rest of the feedback can go in the most recent comment.